ElDorito/Source/Discord/DiscordRPC.cpp

```diff
@@ -76,10 +104,193 @@ namespace
 
 		Web::Ui::ScreenLayer::Notify("discord-joinrequest", jsonBuffer.GetString(), true);
 	}
+
+	void PresenceUpdate()
+	{
+		DiscordRichPresence discordPresence;
+		memset(&discordPresence, 0, sizeof(discordPresence));
+
```

Comment:
You're not using the presence defined here https://github.com/ElDewrito/ElDorito/blob/master/ElDorito/Source/Discord/DiscordRPC.h#L9

By reusing the presence we can avoid nulling it out each time we want to update.